acceptance_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package activemq
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/google/uuid"
+)
+
+func TestAcceptance(t *testing.T) {
+	sourceConfig := map[string]string{
+		"url":              "localhost:61613",
+		"user":             "admin",
+		"password":         "admin",
+		"subscriptionType": "ANYCAST",
+
+		// we want to disable artemis flow control, so that messages are delivered as soon as possible.
+		// This prevents source reads from timing out in unexpected ways.
+		"consumerWindowSize": "-1",
+	}
+	destinationConfig := map[string]string{
+		"url":             "localhost:61613",
+		"user":            "admin",
+		"password":        "admin",
+		"destinationType": "ANYCAST",
+	}
+
+	driver := sdk.ConfigurableAcceptanceTestDriver{
+		Config: sdk.ConfigurableAcceptanceTestDriverConfig{
+			Connector:         Connector,
+			SourceConfig:      sourceConfig,
+			DestinationConfig: destinationConfig,
+			BeforeTest: func(t *testing.T) {
+				destination := uniqueDestinationName(t)
+				sourceConfig["destination"] = destination
+				destinationConfig["destination"] = destination
+				destinationConfig["destinationHeader"] = destination
+			},
+			Skip: []string{
+				// Configure tests are faulty since we rely on paramgen to validate required parameters.
```

Comment:
Hmm, how do you mean faulty? An issue with the connector SDK?

Reply:
Basically the root of the issue ignoring these two is that both `source.Configure` and `destination.Configure` don't validate the given configuration. The idea that I got was that validation with struct tags like `required` or `inclusion=val1|val2`  was done in an upper layer, not by the Configure method.

I'd be happy to validate them again, would prevent me silly mistakes developing acceptance tests, but I don't see any function on the sdk that used the `map[string]sdk.Parameter` to validate.